src/EnergyPlus/HVACVariableRefrigerantFlow.cc

```diff
@@ -14268,14 +14277,17 @@ void VRFCondenserEquipment::VRFOU_CalcCompC(EnergyPlusData &state,
                 MinOutdoorUnitPe = max(P_discharge - this->CompMaxDeltaP, MinRefriPe);
                 MinOutdoorUnitTe = GetSatTemperatureRefrig(
                     state, this->RefrigerantName, max(min(MinOutdoorUnitPe, RefPHigh), RefPLow), RefrigerantIndex, RoutineName);
+                // Te can't be smaller than user input lower bound
+                MinOutdoorUnitTe = max(this->IUEvapTempLow, MinOutdoorUnitTe);
 
                 auto f = [&state, T_discharge_new, CondHeat, CAPFT](Real64 const T_suc) {
                     return CompResidual_FluidTCtrl(state, T_discharge_new, CondHeat, CAPFT, T_suc);
                 };
 
                 General::SolveRoot(state, 1.0e-3, MaxIter, SolFla, SmallLoadTe, f, MinOutdoorUnitTe,
-                                   T_suction);   // SmallLoadTe is the updated Te'
-                if (SolFla < 0) SmallLoadTe = 6; // MinOutdoorUnitTe; //SmallLoadTe( Te'_new ) is constant during iterations
```

Comment:
This is poor logic for a SolveRoot failure. -1 means iteration limit exceeded and -2 means it can't find a solution. Using SolFla < 0 captures both and assumes there is a small load. Maybe this works, maybe it doesn't, I don't know. This is a separate issue, if this is actually an issue.

Reply:
I will separate out SolFla=-1 and SolFla=-2 case. When it is -1 I will throw an error message of not reaching convergence. When it is -2, it would be the case where the load is small. 

In the -2 case, f(MinOutdoorUnitTe) and f(T_suction) can't be both positive as f(T_suction) > 0 would mean 
```
Q_evap_req * C_cap_operation / this->RatedEvapCapacity > CurveValue(state, this->OUCoolingCAPFT(CounterCompSpdTemp), T_discharge, T_suction)
```
, which is 
```
Q_evap_req * C_cap_operation > this->RatedEvapCapacity * CurveValue(state, this->OUCoolingCAPFT(CounterCompSpdTemp), T_discharge, T_suction)
```
If this is the case, it shouldn't have been in `if (Q_evap_req * C_cap_operation <= CompEvaporatingCAPSpd(CounterCompSpdTemp)) {`